src/ui/Controls/NikseComboBox.cs

```diff
@@ -484,78 +489,18 @@ protected override void OnMouseDown(MouseEventArgs e)
                     _listViewShown = true;
                     _textBox.Focus();
 
-                    if (_listView == null)
```

Comment:
This is now moved to a method that will handle the initialization for the `listview`